docker/pypi/wmagent/run.sh

```diff
@@ -1,11 +1,23 @@
 #!/bin/bash
 
 ### Basic initialization wrapper for WMAgent to serve as the main entry point for the WMAgent Docker container
+wmaUser=$(id -un)
+wmaGroup=$(id -gn)
+wmaUserID=$(id -u)
+wmaGroupID=$(id -g)
+export WMA_USER=$wmaUser
+echo "Running WMAgent container with user: $wmaUser (ID: $wmaUserID) and group: $wmaGroup (ID: $wmaGroupID)"
 
+echo "Correcting ownership for WMA_ROOT_DIR: $WMA_ROOT_DIR"
+find $WMA_ROOT_DIR \! \( -user $wmaUser -group $wmaGroup \) -exec chown -f $wmaUser:$wmaGroup '{}' +;
+
+# append the WMAgent user to the mysql group
+if getent passwd mysql >/dev/null 2>&1; then
```

Comment:
Why needed ??????? 
I am pretty sure everything was working before without the need of having the extra step of appending the `WMA__USER`  to the `mysql` group. We are not using sockets inside the container. We are using the standard client, so always connecting to the mysql database through the network interface and the loopback device. If this is needed here, it is a sign to me only for one thing - someone is trying to connect to a local database at the host (possibly a databse from an old agent) through a socket (possibly exposed at the host from the old agent). Which is genuinely wrong. We should not even try this.

And more in very near future we will move to `mariadb` aliases instead of the `mysql` old ones. including the system groups.

Reply:
This logic is already in our dockerfile, see: https://github.com/dmwm/CMSKubernetes/blob/master/docker/pypi/wmagent/Dockerfile#L40

It might be something legacy then, when we considered running MariaDB inside the same image as of WMAgent. I am removing this as well.

Reply:
yep.... that should be legacy.